[2025-02-05T00:28:12.505+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-05T00:28:12.517+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.dm_transaction_type_snapshot manual__2025-02-05T00:00:00+00:00 [queued]>
[2025-02-05T00:28:12.523+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.dm_transaction_type_snapshot manual__2025-02-05T00:00:00+00:00 [queued]>
[2025-02-05T00:28:12.524+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-05T00:28:12.534+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtSnapshotLocalOperator): model_credit_fraud.dm_transaction_type_snapshot> on 2025-02-05 00:00:00+00:00
[2025-02-05T00:28:12.538+0000] {standard_task_runner.py:63} INFO - Started process 1702 to run task
[2025-02-05T00:28:12.540+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_dbt-model_credit_fraud', 'model_credit_fraud.dm_transaction_type_snapshot', 'manual__2025-02-05T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt-model_fraud_credit.py', '--cfg-path', '/tmp/tmp4gb_4lq7']
[2025-02-05T00:28:12.541+0000] {standard_task_runner.py:91} INFO - Job 24: Subtask model_credit_fraud.dm_transaction_type_snapshot
[2025-02-05T00:28:12.579+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.dm_transaction_type_snapshot manual__2025-02-05T00:00:00+00:00 [running]> on host eb20c43aeeb4
[2025-02-05T00:28:12.643+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_dbt-model_credit_fraud' AIRFLOW_CTX_TASK_ID='model_credit_fraud.dm_transaction_type_snapshot' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-05T00:00:00+00:00'
[2025-02-05T00:28:12.645+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-05T00:28:14.518+0000] {local.py:220} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-02-05T00:28:14.519+0000] {local.py:436} INFO - Cloning project to writable temp directory /tmp/tmpluzx65fg from /usr/local/airflow/dags/dbt/localiza_dw
[2025-02-05T00:28:14.522+0000] {local.py:447} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dag_dbt-model_credit_fraud__model_credit_fraud/target/partial_parse.msgpack
[2025-02-05T00:28:14.526+0000] {config.py:348} INFO - Profile caching is enable.
[2025-02-05T00:28:14.532+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2025-02-05T00:28:14.533+0000] {config.py:328} INFO - Profile found in cache using profile: /tmp/cosmos/profile/bdc35343eb23031ee554202b16407e794194afcc614f7ea62bc658c63495a211/profiles.yml.
[2025-02-05T00:28:14.534+0000] {local.py:408} INFO - Trying to run dbtRunner with:
 ['snapshot', '--models', 'dm_transaction_type', '--project-dir', '/tmp/tmpluzx65fg', '--profiles-dir', '/tmp/cosmos/profile/bdc35343eb23031ee554202b16407e794194afcc614f7ea62bc658c63495a211', '--profile', '***', '--target', 'prod']
 in /tmp/tmpluzx65fg
[2025-02-05T00:28:14.548+0000] {logging_mixin.py:188} INFO - 00:28:14  Running with dbt=1.6.1
[2025-02-05T00:28:14.631+0000] {logging_mixin.py:188} INFO - 00:28:14  Registered adapter: postgres=1.6.1
[2025-02-05T00:28:14.800+0000] {logging_mixin.py:188} INFO - 00:28:14  Found 3 models, 1 snapshot, 5 tests, 1 source, 0 exposures, 0 metrics, 349 macros, 0 groups, 0 semantic models
[2025-02-05T00:28:14.802+0000] {logging_mixin.py:188} INFO - 00:28:14
[2025-02-05T00:28:14.848+0000] {logging_mixin.py:188} INFO - 00:28:14  Concurrency: 1 threads (target='prod')
[2025-02-05T00:28:14.849+0000] {logging_mixin.py:188} INFO - 00:28:14
[2025-02-05T00:28:14.852+0000] {logging_mixin.py:188} INFO - 00:28:14  1 of 1 START snapshot dm_credit_operations.dm_transaction_type ................. [RUN]
[2025-02-05T00:28:16.418+0000] {logging_mixin.py:188} INFO - 00:28:16  1 of 1 OK snapshotted dm_credit_operations.dm_transaction_type ................. [success in 1.56s]
[2025-02-05T00:28:16.431+0000] {logging_mixin.py:188} INFO - 00:28:16
[2025-02-05T00:28:16.432+0000] {logging_mixin.py:188} INFO - 00:28:16  Finished running 1 snapshot in 0 hours 0 minutes and 1.63 seconds (1.63s).
[2025-02-05T00:28:16.442+0000] {logging_mixin.py:188} INFO - 00:28:16
[2025-02-05T00:28:16.443+0000] {logging_mixin.py:188} INFO - 00:28:16  Completed successfully
[2025-02-05T00:28:16.444+0000] {logging_mixin.py:188} INFO - 00:28:16
[2025-02-05T00:28:16.445+0000] {logging_mixin.py:188} INFO - 00:28:16  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-02-05T00:28:17.175+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v10.json is above dbt-ol supported version 7. This might cause errors.
[2025-02-05T00:28:17.215+0000] {local.py:494} INFO - Inlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_credit_operations.stg_credit_fraud', extra=None)]
[2025-02-05T00:28:17.215+0000] {local.py:495} INFO - Outlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_credit_operations.dm_transaction_type', extra=None)]
[2025-02-05T00:28:17.215+0000] {local.py:595} INFO - Assigning inlets/outlets without DatasetAlias
[2025-02-05T00:28:17.216+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-02-05T00:28:17.228+0000] {dag.py:3954} INFO - Setting next_dagrun for dag_dbt-model_credit_fraud to None, run_after=None
[2025-02-05T00:28:17.253+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-05T00:28:17.279+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_dbt-model_credit_fraud, task_id=model_credit_fraud.dm_transaction_type_snapshot, run_id=manual__2025-02-05T00:00:00+00:00, execution_date=20250205T000000, start_date=20250205T002812, end_date=20250205T002817
[2025-02-05T00:28:17.312+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-05T00:28:17.329+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-05T00:28:17.331+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-02-05T14:40:58.021+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-05T14:40:58.042+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.dm_transaction_type_snapshot manual__2025-02-05T00:00:00+00:00 [queued]>
[2025-02-05T14:40:58.055+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.dm_transaction_type_snapshot manual__2025-02-05T00:00:00+00:00 [queued]>
[2025-02-05T14:40:58.056+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-05T14:40:58.076+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtSnapshotLocalOperator): model_credit_fraud.dm_transaction_type_snapshot> on 2025-02-05 00:00:00+00:00
[2025-02-05T14:40:58.081+0000] {standard_task_runner.py:63} INFO - Started process 544 to run task
[2025-02-05T14:40:58.087+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_dbt-model_credit_fraud', 'model_credit_fraud.dm_transaction_type_snapshot', 'manual__2025-02-05T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt-model_fraud_credit.py', '--cfg-path', '/tmp/tmp3q7x9h0s']
[2025-02-05T14:40:58.089+0000] {standard_task_runner.py:91} INFO - Job 25: Subtask model_credit_fraud.dm_transaction_type_snapshot
[2025-02-05T14:40:58.156+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.dm_transaction_type_snapshot manual__2025-02-05T00:00:00+00:00 [running]> on host eb17fa5f9c23
[2025-02-05T14:40:58.261+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_dbt-model_credit_fraud' AIRFLOW_CTX_TASK_ID='model_credit_fraud.dm_transaction_type_snapshot' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-05T00:00:00+00:00'
[2025-02-05T14:40:58.265+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-05T14:41:01.478+0000] {local.py:220} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-02-05T14:41:01.479+0000] {local.py:436} INFO - Cloning project to writable temp directory /tmp/tmplny5lpio from /usr/local/airflow/dags/dbt/localiza_dw
[2025-02-05T14:41:01.499+0000] {local.py:447} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dag_dbt-model_credit_fraud__model_credit_fraud/target/partial_parse.msgpack
[2025-02-05T14:41:01.505+0000] {config.py:348} INFO - Profile caching is enable.
[2025-02-05T14:41:01.516+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2025-02-05T14:41:01.518+0000] {config.py:328} INFO - Profile found in cache using profile: /tmp/cosmos/profile/bdc35343eb23031ee554202b16407e794194afcc614f7ea62bc658c63495a211/profiles.yml.
[2025-02-05T14:41:01.519+0000] {local.py:408} INFO - Trying to run dbtRunner with:
 ['snapshot', '--models', 'dm_transaction_type', '--project-dir', '/tmp/tmplny5lpio', '--profiles-dir', '/tmp/cosmos/profile/bdc35343eb23031ee554202b16407e794194afcc614f7ea62bc658c63495a211', '--profile', '***', '--target', 'prod']
 in /tmp/tmplny5lpio
[2025-02-05T14:41:01.539+0000] {logging_mixin.py:188} INFO - 14:41:01  Running with dbt=1.6.1
[2025-02-05T14:41:01.675+0000] {logging_mixin.py:188} INFO - 14:41:01  Registered adapter: postgres=1.6.1
[2025-02-05T14:41:02.458+0000] {logging_mixin.py:188} INFO - 14:41:02  Found 4 models, 1 snapshot, 4 seeds, 5 tests, 3 sources, 0 exposures, 0 metrics, 349 macros, 0 groups, 0 semantic models
[2025-02-05T14:41:02.466+0000] {logging_mixin.py:188} INFO - 14:41:02
[2025-02-05T14:41:02.571+0000] {logging_mixin.py:188} INFO - 14:41:02  Concurrency: 1 threads (target='prod')
[2025-02-05T14:41:02.573+0000] {logging_mixin.py:188} INFO - 14:41:02
[2025-02-05T14:41:02.577+0000] {logging_mixin.py:188} INFO - 14:41:02  1 of 1 START snapshot dm_credit_operations.dm_transaction_type ................. [RUN]
[2025-02-05T14:41:05.096+0000] {logging_mixin.py:188} INFO - 14:41:05  1 of 1 OK snapshotted dm_credit_operations.dm_transaction_type ................. [success in 2.51s]
[2025-02-05T14:41:05.120+0000] {logging_mixin.py:188} INFO - 14:41:05
[2025-02-05T14:41:05.122+0000] {logging_mixin.py:188} INFO - 14:41:05  Finished running 1 snapshot in 0 hours 0 minutes and 2.64 seconds (2.64s).
[2025-02-05T14:41:05.138+0000] {logging_mixin.py:188} INFO - 14:41:05
[2025-02-05T14:41:05.140+0000] {logging_mixin.py:188} INFO - 14:41:05  Completed successfully
[2025-02-05T14:41:05.142+0000] {logging_mixin.py:188} INFO - 14:41:05
[2025-02-05T14:41:05.143+0000] {logging_mixin.py:188} INFO - 14:41:05  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-02-05T14:41:05.931+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v10.json is above dbt-ol supported version 7. This might cause errors.
[2025-02-05T14:41:05.977+0000] {local.py:494} INFO - Inlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_credit_operations.stg_credit_fraud', extra=None)]
[2025-02-05T14:41:05.978+0000] {local.py:495} INFO - Outlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_credit_operations.dm_transaction_type', extra=None)]
[2025-02-05T14:41:05.979+0000] {local.py:595} INFO - Assigning inlets/outlets without DatasetAlias
[2025-02-05T14:41:05.979+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-02-05T14:41:05.994+0000] {dag.py:3954} INFO - Setting next_dagrun for dag_dbt-model_credit_fraud to None, run_after=None
[2025-02-05T14:41:06.027+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-05T14:41:06.059+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_dbt-model_credit_fraud, task_id=model_credit_fraud.dm_transaction_type_snapshot, run_id=manual__2025-02-05T00:00:00+00:00, execution_date=20250205T000000, start_date=20250205T144058, end_date=20250205T144106
[2025-02-05T14:41:06.110+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-05T14:41:06.128+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-05T14:41:06.131+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
